addons/product/models/product_template.py

```diff
@@ -403,6 +403,11 @@ def create(self, vals_list):
         return templates
 
     def write(self, vals):
+        if (('type' in vals and vals['type'] != 'service') or ('landed_cost_ok' in vals and vals['landed_cost_ok'] == False)) and self.type == 'service' and self.landed_cost_ok:
```

Comment:
```suggestion
        if (('type' in vals and vals['type'] != 'service') or ('landed_cost_ok' in vals and vals['landed_cost_ok'] == False)) and self.type == 'service' and hasattr(self, 'landed_cost_ok') and self.landed_cost_ok:
```
AFAIK landed_cost_ok is declared in `stock_landed_cost` module, not in `product` module.

Reply:
Yes indeed, thanks @MiquelRForgeFlow for the review!